lua/kulala/parser/curl.lua

```diff
@@ -89,7 +92,7 @@ function M.parse(curl)
         res.headers["user-agent"] = arg
       elseif state == State.Header then
         local header, value = Stringutils.cut(arg, ":")
-        res.headers[Stringutils.remove_extra_space(header)] = Stringutils.remove_extra_space(value)
+        res.headers[Stringutils.remove_extra_space(header):lower()] = Stringutils.remove_extra_space(value)
```

Comment:
I don't think we can do that, because that would be a regression to stuff that @Grueslayer fixed, right?

Reply:
Should be fixed in 6760d2c